actionpack/lib/action_dispatch/http/permissions_policy.rb

```diff
@@ -23,13 +23,16 @@ module ActionDispatch # :nodoc:
   #
   class PermissionsPolicy
     class Middleware
-      CONTENT_TYPE = "Content-Type"
       # The Feature-Policy header has been renamed to Permissions-Policy.
       # The Permissions-Policy requires a different implementation and isn't
       # yet supported by all browsers. To avoid having to rename this
       # middleware in the future we use the new name for the middleware but
       # keep the old header name and implementation for now.
-      POLICY       = "Feature-Policy"
+      if Gem::Version.new(Rack::RELEASE) >= Gem::Version.new("3")
+        POLICY = "feature-policy"
+      else
+        POLICY = "Feature-Policy"
+      end
```

Comment:
Since it looks like this type of check will be necessary for multiple headers spread across files, I wonder if we should create some undocumented method like `ActionDispatch.header_for_current_rack` that we could use like: `POLICY = ActionDispatch.header_for_current_rack("Feature-Policy")`